dhall-nix/exec/Main.hs

```diff
@@ -48,14 +48,20 @@ main = handle (Dhall.detailed (do
         Left  err  -> Control.Exception.throwIO err
         Right expr -> return expr
 
-    expr' <- Dhall.Import.load expr
-    case Dhall.TypeCheck.typeOf expr' of
-        Left  err -> Control.Exception.throwIO err
-        Right _   -> return ()
+    -- expr' <- Dhall.Import.load expr
```

Comment:
I commented this out since I need to do remote URL imports within Nix.

However, it would be nice if I could use some function in `Dhall.Import` to only do local imports, but leave all remote URL imports.  I poked around in `Dhall.Import` to see if this was possible, but I couldn't really come up with anything.

Is there some easy way to do this?  Or should I just handle local imports in Nix as well?